docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1529,6 +1529,17 @@ simulants living in military group quarters we will set their employer
 to Military, and ensure that their address and zip code match their
 employer_address and employer_zipcode.
 
+Speaking of the names for employers, we will generate names for
+employers based on a conditional random model that Abie developed and
+Jim refactored into the vivarium model.  It should have more details
+documented at some point.  Jim will ensure that all employer names are
+unique for now, although it would be interesting one day to add some
+duplicates.  Businesses can have the same name in real life but
+if/when we do have duplicate names we need to prevent duplicates for
+certain most common names.
```

Comment:
What do you mean by this? No duplicates for the biggest employers?

Reply:
to be determined, but for example, there should perhaps only be one Postal Service (https://en.wikipedia.org/wiki/The_Postal_Service#United_States_Postal_Service_controversy)

Reply:
Makes sense, but I don't clearly understand that meaning from what you have written here -- how can it be "common" if there is only 1 business with that name?

That's why I was proposing that it should be "... we need to prevent duplicates of the names of the largest employers." Your USPS example makes me think I got that right, but I'm not sure.

Reply:
I agree with Zeb on this phrasing ^^